log4j-core/src/test/java/org/apache/logging/log4j/core/AppenderRefLevelTest.java

```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+
```

Comment:
Static imports go last. This fix is incorrect. We also allow wildcard static imports so this change isn't required.